### ISSUE TITLE
Bind failed: port is already allocated. 

### ISSUE THREAD

**Issue by MollyBe** [Cloudbox/Community#95]
After running the running radarrx role as a normal community role I got this error:

>fatal: [localhost]: FAILED! => {"changed": false, "msg": "Error starting container [altered-random-string]: 
500 Server Error: Internal Server Error (\"driver failed programming external connectivity on endpoint [altered-container-name] ([altered-random-string]): Bind for 127.0.0.1:17882 failed: port is already `allocated\")"}

Steps to reproduce:

Run the raddarrx role with 3 options,

```
radarrx:
  roles: [role1, role2. role3]
```

Add an extra role / change the roles and run it again.

```
radarrx:
  roles: [role1, extra1, role2.]
```

Solution, stop docker role2 and perhaps all other raddarx dockers before running the community tag.

**Comment by maximuskowalski** [comment]
You can't remove roles and then run it with new options. Any roles you wish to use / keep should be left in situ.
```
radarrx:
  roles: [role1, role2. role3]
```
 should become
```
radarrx:
  roles: [role1, role2. role3, extra1
```

**Comment by desimaniac** [comment]
`Solution, stop docker role2 and perhaps all other raddarx dockers before running the community tag.`

That seems like a good idea.

**Comment by Migz93** [comment]
We could apply labels to the various containers when deploying them. Then as part of the role we stop (or remove) any containers with those labels?
Similiar to how main CB backup only stops containers with the "com.github.cloudbox.cloudbox_managed" label?
If no one sees an issue with doing it that way I'll see if i can implement it this weekend.

**Comment by MollyBe** [comment]
Perhaps stopping and removing the containers would be the best way to proceed.

If we only stopped all the containers I can foresee an issue in this scenario. 

Say the user changed settings.yml from
_radarrx:
  roles: [role1, role2. role3]_
to
_radarrx:
  roles: [roleA, role2. role3]_

role1 would still exist as a container and now have the same port mapping as the new roleA creating a conflict.

**Comment by desimaniac** [comment]
@Migz93 

something like ...
```
"com.github.cloudbox.community": radarrx
```

**Comment by Migz93** [comment]
Resolved this issue with #107 .
As discussed above, added a label to be deployed with all arrX containers and then when running the role it will remove & stop all containers with that label.
